static/js/app.js

```diff
@@ -178,30 +69,30 @@ async function addLifeEvents() {
       ${event.more ? `<p>More: ${event.more}</p>` : ''}
     </div>
   `).join('');
-  console.log("life events",lifeEventsHtml)
+  console.log(`${eventType} events`, eventsHtml)
   // Append the generated HTML to the container
-  lifeEventsContainer.innerHTML = lifeEventsHtml;
+  eventsContainer.innerHTML = eventsHtml;
 }
 
 ////////////////////////Execute methods\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
-//DOM Listener makes sure all the content is loaded befere execute the methods
+//DOM Listener makes sure all the content is loaded before execute the methods
 document.addEventListener('DOMContentLoaded', async () => {
-  // loads About me page
-  await loadAboutMePage();
```

Comment:
It looks like you forgot to delete the definition of the function `loadAboutMePage`. Since you deleted its only use, now this function should be removed as well.